utils/models.py

```
@@ -0,0 +1,178 @@
+        with torch.no_grad():
+            return self.model(**tokens)
+
+
+class PoliticalBiasBert(Model):
+    def __init__(self):
+        super().__init__(
+            AutoTokenizer.from_pretrained("bert-base-cased"),
+            AutoModelForSequenceClassification.from_pretrained(
+                "bucketresearch/politicalBiasBERT"
+            ),
+            True,
```

Ruff

Boolean positional value in function call

(FBT003)